integration-tests/prometheus-java-client-expected-changes.patch

```diff
@@ -4565,15 +4603,24 @@
  }
 --- a/prometheus-metrics-exporter-opentelemetry/src/test/java/io/prometheus/metrics/exporter/opentelemetry/otelmodel/PrometheusMetricDataTest.java
 +++ b/prometheus-metrics-exporter-opentelemetry/src/test/java/io/prometheus/metrics/exporter/opentelemetry/otelmodel/PrometheusMetricDataTest.java
-@@ -7,7 +7,7 @@ import io.prometheus.metrics.model.snapshots.Unit;
+@@ -3,14 +3,15 @@ package io.prometheus.metrics.exporter.opentelemetry.otelmodel;
+ import static java.util.Map.entry;
+ import static org.junit.jupiter.api.Assertions.*;
+ 
++import com.google.common.collect.ImmutableMap;
+ import io.prometheus.metrics.model.snapshots.Unit;
  import java.util.Map;
  import org.junit.jupiter.api.Test;
  
 -class PrometheusMetricDataTest {
 +final class PrometheusMetricDataTest {
  
    Map<Object, Object> translations =
-       Map.ofEntries(
+-      Map.ofEntries(
++      ImmutableMap.ofEntries(
+           entry("days", "d"),
+           entry("hours", "h"),
+           entry("minutes", "min"),
```

Comment:
This isn't simplified further, because it's a _long_ list. We could add more rules, but: meh.